packages/node_modules/pouchdb-adapter-indexeddb/src/getAttachment.js

```diff
@@ -2,9 +2,16 @@
 
 import { btoa, readAsBinaryString } from 'pouchdb-binary-utils';
 
-import { DOC_STORE } from './util';
+function getAttachment(txn, docId, attachId, _, opts, cb) {
+  if (txn.error) {
+    return cb(txn.error);
+  }
+
+  const doc = opts.metadata;
+  const rev = doc.revs[opts.rev || doc.rev].data;
+  const digest = rev._attachments[attachId].digest;
```

Comment:
Just a thought: Why did you need to lookup the digest from doc.revs?
The fourth parameter `_` already has the corresponding attachment-data (refer [core/adapter.js:704](https://github.com/pouchdb/pouchdb/blob/c5e103e32f0ce4e8e510726aaa867d3479665615/packages/node_modules/pouchdb-core/src/adapter.js#L704))

Reply:
I just copied the existing code :grin: I'll give that fourth parameter a try, thanks!

Reply:
Testing at https://github.com/alxndrsn/pouchdb/actions/runs/6460505912

Reply:
@SourceR85 great suggestion, thanks :+1: and it looks like it works :grin: